contexts/newsroom.tsx

```diff
@@ -31,3 +31,7 @@ export const NewsroomContextProvider: FunctionComponent<Context> = ({
         {children}
     </NewsroomContext.Provider>
 );
+
+NewsroomContextProvider.defaultProps = {
+    selectedCategory: undefined,
```

Comment:
Do we really need `defaultProps` for `undefined` values? I believe they'd undefined either way when not provided.

Reply:
Well, we need it for the `react/require-default-props` ESLint rule.

However I believe this rule is redundant since we're using Typescript which is gonna warn us if we try to access a prop that might not have a value.

So I vote for disabling the `react/require-default-props` rule entirely and remove the `defaultProps` declaration in this PR.

Reply:
Yeah, I also think this rule is redundant. Let's remove it!